### PR TITLE
75 as a player i want 5 ware types, and 3 cities

### PR DESCRIPTION
**Pull request by knoffi**

### CODE REVIEW

File: src/logic/Ware.config.ts [hunk 1]
```diff
@@ -5,10 +5,14 @@ export const wareConfig = {
         [WareType.Furs]: 355,
         [WareType.Juwelry]: 670,
         [WareType.Salt]: 48,
+        [WareType.Fish]: 10,
```
Comment (proSingularity) [inline src/logic/Ware.config.ts]: The tests assume that `maxPrice` is strictly larger than `minprice`. They also failed on Travis. I suppose you forgot to run the tests?

Comment (proSingularity) [reply src/logic/Ware.config.ts]: same for beer

File: src/scenes/TableScene.ts [hunk 2]
```diff
@@ -86,6 +91,18 @@ export class TableScene extends Scene {
         text.init(this.logic, ware);
     }
 
+    private addQuantityButton(x: number, y: number, quantity: number) {
+        this.quantityButton = new QuantityButton(
```
Comment (proSingularity) [inline src/scenes/TableScene.ts]:
```suggestion
        const quantityButton = new QuantityButton(
```

Comment (proSingularity) [reply src/scenes/TableScene.ts]: There is no need to make this a data field. In fact, you overwrite the data field twice. Also delete the data field.

File: src/logic/Logic.ts [hunk 3]
```diff
@@ -29,6 +29,10 @@ export class Logic implements ILogic {
         this.selectedCity = startCity;
     }
 
+    public setTradedQuantity(quantity: number) {
```
Comment (proSingularity) [inline src/logic/Logic.ts]: Please add tests for the newly added method.

Alternatively, simply make `tradedQuantity` `public`. In this case, the data field is very simple (`tradedQuantity: number`). It does not have a complicated representation that we want to hide from its users.. setter and getter only add overhead. 

FYI: An example where getter and setter are more needed would be a graph. There are many equivalent implementations of a  graph, and if we wanted to abstract from the concrete implementation we chose, we could add several getter and setter, each for different implementations of the graph.

File: src/logic/Ware.config.ts (hunk 1 above)
Comment (knoffi) [reply src/logic/Ware.config.ts]: I fixed the price boundaries and the tests have all passed. Is there an easy way to fix the ''two issues in one pull request''? I overlooked it...

**Pull request merged**